testsuite/cluster-test/src/main.rs

```diff
@@ -172,12 +178,14 @@ pub fn main() {
     let mut runner = ClusterTestRunner::setup(&args);
 
     if let Some(ref hash_or_tag) = args.deploy {
-        // Deploy deploy_hash before running whatever command
-        let hash = runner
-            .deployment_manager
-            .resolve(hash_or_tag)
-            .expect("Failed to resolve tag");
-        exit_on_error(runner.redeploy(&hash));
+        if !runner.k8s {
```

Comment:
Looks like with k8s we deploy every time, previously we were able to keep cluster as is and run additional commands on it.

Is it hard to preserve same thing for k8s setup? Discover cluster if no `--deploy` provided or deploy it if this command is present?

Reply:
We can do discovery similar to AWS, but I don't see any immediate use cases for it right now. We can do it later.